catalog/gke/subnet/subnet.yaml

```diff
@@ -0,0 +1,32 @@
+# A subnet designed for use by a single GKE cluster.
+# Capacity:
+# ~1k nodes (/22)
+# ~65k pods (/16)
+# ~65k services (/16)
+# Recommended pods per node: 64
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSubnetwork
+metadata:
+  name: platform-project-id-example-us-east4 # {"$kpt-set":"subnet-name"}
+  # Landing Zone networks & subnets are managed in the networking namespace
+  namespace: networking # {"$kpt-set":"networking-namespace"}
+  annotations:
+    cnrm.cloud.google.com/project-id: network-project-id # {"$kpt-set":"network-project-id"}
+spec:
+  description: platform-project-id-example-us-east4 # {"$kpt-set":"subnet-name"}
+  region: us-east4 # {"$kpt-set":"location"}
+  networkRef:
+    external: projects/network-project-id/global/networks/default # {"$kpt-set":"network-selflink"}
+  ipCidrRange: 10.3.4.0/22 # {"$kpt-set":"node-ip-range"}
+  secondaryIpRange:
+  - ipCidrRange: 172.17.0.0/16 # {"$kpt-set":"pod-ip-range"}
+    rangeName: pods
+  - ipCidrRange: 172.18.0.0/16 # {"$kpt-set":"service-ip-range"}
+    rangeName: services
+  privateIpGoogleAccess: true
+  # Uncomment to enable VPC Flow Logs
```

Comment:
Would this be discoverable? Wondering if it would be better to point users to this via docs

Reply:
This is another one of those optional feature problems...
In this case, I didn't need it for Tenant Factory, so I just left it commented as a hint of what's possible.
I don't really want to document every possible optional config field...

Reply:
While I think many people will want these, I'm just gonna remove all these optional and commented out features.

The README links to the KCC reference for each resource in the blueprint.

Reply:
Removed.